### PR TITLE
Fix hwinfo

### PR DESCRIPTION
**Pull request by mkosola**

### CODE REVIEW

File: rpm/testrunner-lite.spec [hunk 1]
```diff
@@ -16,7 +16,7 @@ BuildRequires: libuuid-devel
 # libxml2 and libcurl are implicit dependencies  
 Requires: test-definition
 Requires: openssh
-Requires: %{name}-hwinfo-nemo = %{version}-%{release}
+Requires: %{name}-hwinfo-sfos = %{version}-%{release}
 Requires: libuuid
```
Comment (saukko) [inline rpm/testrunner-lite.spec:20]: Why is this libuuid here? Shouldn't it come automaticallyl as requiremenet?

File: rpm/testrunner-lite.spec [hunk 2]
```diff
@@ -39,21 +39,24 @@ Summary: Provides commands for hardware information obtaining
 Requires: coreutils
 Provides: %{name}-hwinfo
 Conflicts: %{name}-hwinfo-meego
-Conflicts: %{name}-hwinfo-nemo
+Conflicts: %{name}-hwinfo-sfos
 
 %package hwinfo-meego
```
Comment (saukko) [inline rpm/testrunner-lite.spec:44]: Would cut out meego and maemo at this point :)

**Review by saukko — COMMENTED** [review]
Also %clean from .spec could be cut out.

**Comment by saukko** [comment]
Also would use sailfish instead of sfos in the naming as we haven't used sfos really in anything so far so lets not invent new naming.

File: rpm/testrunner-lite.spec (hunk 2 above)
Comment (mkosola) [reply rpm/testrunner-lite.spec]: Don't know, dropped it.

**Review by saukko — APPROVED** [review]
LGTM

**Pull request merged**